usecases.liquid

```diff
@@ -0,0 +1,130 @@
+	#virto-logo svg path { fill: var(--clr-lavender-lightest); }
+
+	body { background-color: var(--clr-olive-darker); }
+	#cubo-intro { justify-content: space-evenly; align-items: center; gap: 2rem; padding-bottom: 0; }
+	.text-section { max-width: 500px; }
+	.text-section h5 { line-height: 1.5; }
+	figure.logo svg {
+    	width: clamp(250px, 35vw, 800px);
+	}
+
+	#cubo-intro figure.lavender-arrow svg { width: clamp(200px, 25vw, 300px); }
+
+	#cubo-intro .lavender-arrow { position: relative; left: 16%; top: -45px;}
+
+	#cubo {
+		align-items: center;
+		gap: 2em;
+		padding-top: 0;
+	}
+
+	figure.hand svg {
+    	width: clamp(250px, 35vw, 400px);
+	}
+
+	.hand {
+		float: left;
+		position: relative;
+		left: 0;
+		margin-left: -3.5em;
+		margin-right: 5em;
+	}
+
+	.text-container { max-width: 670px; }
+	.text-container > *:not(h2) { margin-top: 2em; }
+	.text-container h3 { line-height: 1.5; }
+	h2.bold.light-green-text { margin: 0 2em; margin-bottom: 1em; }
+
+	#cubo-benefits { max-height: fit-content; flex-direction: column; }
+	#cubo-benefits ul { list-style: none; display: flex; gap: 1em; justify-content: center; flex-wrap: wrap; align-items: center; margin-top: 2em; }
+	#cubo-benefits ul li { display: flex; flex-direction: column; flex: 30%; height: 300px; padding: 1.5em; text-align: center; align-content: center; gap: 1em;}
+	i { font-style: normal; font-size: 2em; }
+
+	@media (max-width: 700px) {
+		section { flex-direction: column; }
+		#cubo-intro { flex-direction: column-reverse; }
+		.text-container > * { margin: .5em 0; }
+		#cubo-intro .lavender-arrow { position: relative; left: 18%; }
+		.cube { align-self: center; order: 3; }
+		h2.bold.light-green-text { margin: 0 1em; margin-bottom: 1em; }
+	}
+
+	#cubo-description { gap: 2em; justify-content: space-evenly;}
+	figure.cube svg {
+    	width: clamp(250px, 35vw, 400px);
+	}
+
+	#CTA {
+		display: flex;
+		align-items: center;
+		flex-direction: column;
+		gap: 2em;
+		padding: 3em 1em;
+	}
+	#CTA h3 { max-width: 700px; }
+
+
+</style>
+
+<section id="cubo-intro">
+	<div class="text-section">
+		<h1 class="light-green-text regular">Casos de uso</h1>
+		<h3 class="lightest-lavender-text regular">Conoce casos reales donde las herramientas de Virto han logrado resolver problemas del día a día.</h3>
+		<a class="lightest-lavender-text bolder" href="#cubo">Quiero saber más</a>
+		<figure class="lavender-arrow">
+			{% include "arrow_down_lavender.svg" %}
+	</figure>
+	</div>
+	<figure class="logo">
+			{% include "website.svg" %}
+	</figure>
+</section>
+	
+<section id="cubo" class="light-green-text">
+	<figure class="logo hand">
+			{% include "hand.svg" %}
+	</figure>
+	<div class="text-container">
+		<h2 class="regular">Cubo</h2>
+		<h3 class="regular">Con las herramientas que proporciona Virto, se ha desarrollado <strong class="dark-lavender-text">Cubo</strong>, una plataforma en la que puedes invertir en propiedades fraccionadas de forma fácil y accesible.</h3>
```

Comment:
```suggestion
		<h3 class="regular">Con las herramientas que proporciona Virto, se ha desarrollado <a href="https://cubo.land" class="dark-lavender-text">Cubo</a>, una plataforma en la que puedes invertir en propiedades fraccionadas de forma fácil y accesible.</h3>
```

Reply:
Looks good in general, it's quite CUBO focused at the moment but we can change it overtime to talk about more use cases and how people can create their own use-case(a.k.a. commercial communities)

Reply:
I'm currently working on the design for the BLOQUE section. I agree with that, I'll share your suggestion to the communication team for content creation :)